### PR TITLE
InfluxDB get started

### LINKED ISSUE #1678: Install influx CLI
Not easy to find how to install influx CLI.

- On Get started page:
   - Add keywords for installing the influx CLI.
   - In both Cloud and OSS sections, add overview links.
- Run docsearch.

### LINKED ISSUE #4266: Add links to Configuration instructions to Install Page and Get Started page
Add links to Configuration instructions to both the Install Page and the Get Started page.

For the Install page something along the lines of "For more information on configuration see https://docs.influxdata.com/influxdb/v2.2/reference/config-options/ "

We could also add a "Configure" Bullet item/Link to the Get Started page which links to the Configure page
https://docs.influxdata.com/influxdb/v2.2/get-started/

New user feedback!
Page: https://docs.influxdata.com/influxdb/v2.2/get-started/
Feedback: Where is configuration instructions

### LINKED ISSUE #4409: Add more explanation to Get Started
Get Started lists tools by category (Write, Query, ...), but could be a more explanatory guide.
https://docs.influxdata.com/influxdb/v2.4/get-started/

##### Relevant URLs
- https://influxdata.slack.com/archives/C03HRU09PHA/p1661696401120529

### LINKED ISSUE #4516: Provide more step-by-step instruction in getting started doc
Page: https://docs.influxdata.com/influxdb/v2.4/get-started/
Feedback: Too much nesting

- Remove sections at bottom of page for Query data, Process data, Visualize data, and Monitor and alert on this page (we can link directly to these docs within a step-by-step procedure).

After you’ve [installed InfluxDB OSS](https://docs.influxdata.com/influxdb/v2.4/install/), you're ready to start InfluxDB and collect and write data. 

1. In terminal, start `influxd`.
2. Choose the way you want to collect data (tabs?)--we can refer folks to other docs as alternative ways to collect data, but think it'd be helpful to provide explicit steps for at least one way in this doc, maybe, how to set up the most common Telegraf input.
3. Verify your data is being collected by doing one of the following:
    - Query data (include specific steps to retrieve data collected from the plugin). For more info, see how to Query data.
    - Visualize data (include steps to open a graph/table data in Explorer) For more info, see how to Visualize data.
4. To start processing and monitoring your data, do the following:
     - Create a task to downsample your data (provide steps in this doc)
     - Create an alert to monitor downsampled data (provide steps in this doc)
     For more information, continue to explore how to process and monitor data <insert links>.

### LINKED ISSUE #4522: Add/view credentials in get started
Page: https://docs.influxdata.com/influxdb/v2.4/get-started/
Feedback: what is user and password initially?

Note when the initial username/password is set, and how to access or change

### PR DESCRIPTION
**Pull request by sanderson**
This is the first of two pull requests and includes two pages with only content outlines. Those will be fleshed out in a separate PR into this branch. This PR will not be merged until the second PR is reviewed and merged into this branch.

The goal of this is to completely overhaul the getting started content for InfluxDB and walk users step-by-step the basics of working with InfluxDB as well as foundational concepts. There is a lot here 😬.

As you're reviewing, please let me know if you feel like things are unnecessary or too much. This should be simple for new users to follow, no matter their coding skill.

**Note:** Right now, these changes are only in 2.4, but once everything is reviewed, I'll port the new content to 2.5.

Closes #4522 
Closes #4516 
Closes #4409 
Closes #4266 
Closes #1678

### CODE REVIEW

File: content/influxdb/v2.4/get-started/setup.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+          - **orgID**: [InfluxDB organization ID](/influxdb/v2.4/organizations/view-orgs/#view-your-organization-id)
+          - **type**: Resource type.
+            _For information about what InfluxDB resource types exist, use the
+            [`/api/v2/resources` endpoint](/influxdb/v2.4/api/#operation/GetResources)._
+
+{{% truncate %}}
+```sh
+export INFLUX_HOST=http://localhost:8086
+export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
+export INFLUX_TOKEN=<YOUR_INFLUXDB_OPERATOR_TOKEN>
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:135]: These are already set for OSS by the setup process.

File: content/influxdb/v2.4/get-started/setup.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+  - **permissions**: Array of objects where each object represents permissions
+    for an InfluxDB resource. Each permission contains the following properties:
+      - **action**: "read" or "write"
+      - **resource**: JSON object that represents the InfluxDB resource to grant
+        permission to. Each resource contains at least the following properties:
+          - **orgID**: [InfluxDB organization ID](/influxdb/v2.4/organizations/view-orgs/#view-your-organization-id)
+          - **type**: Resource type.
+            _For information about what InfluxDB resource types exist, use the
+            [`/api/v2/resources` endpoint](/influxdb/v2.4/api/#operation/GetResources)._
+
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:130]: The following example shows how to use cURL with the InfluxDB API to generate an all access token:

File: content/influxdb/v2.4/get-started/setup.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+      {"action": "write", "resource": {"orgID": "'"$INFLUX_ORG_ID"'", "type": "remotes"}},
+      {"action": "read", "resource": {"orgID": "'"$INFLUX_ORG_ID"'", "type": "replications"}},
+      {"action": "write", "resource": {"orgID": "'"$INFLUX_ORG_ID"'", "type": "replications"}}
+    ]
+  }
+'
+```
+{{% /truncate %}}
+
+The request returns a JSON object with the following properties:
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:196]:
```suggestion
The response body contains a JSON object with the following properties:
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
+- **id**: API Token ID
+- **token**: API Token ({{< req "Important" >}})
+- **status**: Token status
+- **description**: Token description
+- **orgID**: InfluxDB organization ID the token is associated with
+- **org**: InfluxDB organization name the token is associated with
+- **userID**: User ID the token is associated with
+- **user**: User ID the token is associated with
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:205]:
```suggestion
- **user**: Username the token is associated with
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+- **permissions**: List of permissions and resources
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:206]:
```suggestion
- **permissions**: List of permissions for organization resources
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+
+**Copy the generated `token` and store it for safe keeping.**
+
+<!------------------------------ END API CONTENT ------------------------------>
+{{% /tab-content %}}
+    {{< /tabs-wrapper >}}
+
+    {{% note %}}
+We recommending using a password manager or a secret store to securely store
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:215]:
```suggestion
We recommend using a password manager or a secret store to securely store
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+sensitive tokens.
+    {{% /note %}}
+
+5.  {{< req text="(Optional)" color="magenta" >}} **Create a bucket**
+
+    In the InfluxDB initialization process, you created a bucket.
+    You can use that bucket or create a new one specifically for this getting
+    started tutorial. All examples in this tutorial assume a bucket named
+    **get-started**.
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:224]:
```suggestion
    _get-started_.
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 8]
```diff
@@ -0,0 +1,332 @@
...
+    ```sh
+    influx bucket create --name get-started
+    ```
+
+<!------------------------------ END CLI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!----------------------------- BEGIN API CONTENT ----------------------------->
+
+To create a new bucket using the InfluxDB HTTP API, send a request to
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:274]:
```suggestion
To create a bucket using the InfluxDB HTTP API, send a request to
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+the InfluxDB API `/api/v2/buckets` endpoint using the `POST` request method.
+
+{{< api-endpoint endpoint="http://localhost:8086/api/v2/buckets" method="post" >}}
+
+Include the following with your request:
+
+- **Headers**:
+  - **Authorization**: Token <INFLUX_TOKEN>
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:282]:
```suggestion
  - **Authorization**: Token `INFLUX_TOKEN`
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 10]
```diff
@@ -0,0 +1,332 @@
...
+  - **Content-Type**: application/json
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:283]:
```suggestion
  - **Content-Type**: `application/json`
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 11]
```diff
@@ -0,0 +1,332 @@
...
+- **Request body**: JSON object with the following properties:
+  - **org**: InfluxDB organization name
+  - **name**: Bucket name
+  - **retentionRules**: List of retention rule objects that define the bucket's retention period.
+    Each retention rule object has the following properties:
+    - **type**: "expire"
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:289]:
```suggestion
    - **type**: `"expire"`
```

File: content/influxdb/v2.4/get-started/setup.md [hunk 12]
```diff
@@ -0,0 +1,332 @@
...
+    - **everySeconds**: Retention period duration in seconds.
+      `0` indicates the retention period is infinite. 
+
+```sh
+export INFLUX_HOST=http://localhost:8086
+export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
+export INFLUX_TOKEN=<YOUR_INFLUXDB_API_TOKEN>
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:296]: These are already set for OSS by the setup process, but if we're telling them to set INFLUX_TOKEN again, should we specify the all access token they just created or leave it to them to decide?

File: content/influxdb/v2.4/get-started/setup.md [hunk 13]
```diff
@@ -0,0 +1,332 @@
...
+
+curl --request POST \
+"$INFLUX_HOST/api/v2/buckets" \
+  --header "Authorization: Token $INFLUX_TOKEN" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "orgID": "'"$INFLUX_ORG_ID"'",
+    "name": "get-started-test",
```
Comment (jstirnaman) [inline content/influxdb/v2.4/get-started/setup.md:304]:
```suggestion
    "name": "get-started",
```

File: layouts/shortcodes/cli/influx-creds-note.html [hunk 14]
```diff
@@ -3,7 +3,12 @@
   <h4 id="authentication-credentials">Authentication credentials</h4>
   <p>
     The examples below assume your InfluxDB <strong>host</strong>, <strong>organization</strong>, and <strong>token</strong> are
-    provided by the <a href="/influxdb/{{ $currentVersion }}/reference/cli/influx/#provide-required-authentication-credentials">active <code>influx</code> CLI configuration</a>.
-    If you do not have a CLI configuration set up, use the appropriate flags to provide these required credentials.
+    provided by either the <a href="/influxdb/{{ $currentVersion }}/reference/cli/influx/#provide-required-authentication-credentials">active <code>influx</code> CLI configuration</a> or by environment variables (<code>INFLUX_HOST</code>, <code>INFLUX_ORG</code>, and <code>INFLUX_TOKEN</code>).
+    If you do not have a CLI configuration set up or the environment variables set, include these required credentials for each command with the following flags:
```
Comment (jstirnaman) [inline layouts/shortcodes/cli/influx-creds-note.html:7]: Could use a similar not for the API examples?

**Review by jstirnaman — APPROVED** [review]
Small fixes and suggestions.

File: content/influxdb/v2.4/get-started/_index.md [hunk 15]
```diff
@@ -0,0 +1,118 @@
...
+Examples of time series data include:
+
+- Industrial sensor data
+- Server performance metrics
+- Heartbeats per minute
+- Electrical activity in the brain
+- Rainfall measurements
+- Stock prices
+
+This multi-part tutorial walks through writing time series data to InfluxDB {{< current-version >}},
```
Comment (lwandzura) [inline content/influxdb/v2.4/get-started/_index.md:29]: I would say "walks you through" I think that's more common

File: content/influxdb/v2.4/get-started/query.md [hunk 16]
```diff
@@ -0,0 +1,597 @@
...
+        column dropdown menu, and then select the **home** measurement.
+    3.  _(Optional)_ To query a specific field or fields, in the next **filter**
+        column, select **_field** from the column dropdown menu, and then
+        select the fields to query. In this tutorial, there are three
+        fields: **co**, **hum**, and **temp**.
+    4.  _(Optional)_ To query by specific tag values, in the next **filter**
+        column, select then tag column from the column dropdown menu, and then
+        select the tag values to filter by. In this tutorial, the tag only
+        tag column is **room**.
+    4.  _(Optional)_ In the **{{% caps %}}Aggregate Function{{% /caps %}}** pane,
```
Comment (lwandzura) [inline content/influxdb/v2.4/get-started/query.md:176]: in hugo server the numbering looks right, not sure why there are two 4s here

File: content/influxdb/v2.4/get-started/write.md [hunk 17]
```diff
@@ -0,0 +1,391 @@
...
+  - /influxdb/v2.4/reference/syntax/line-protocol/
+  - /{{< latest "telegraf" >}}/
+---
+
+InfluxDB provides many different options for ingesting or writing data, including
+the following:
+
+- Influx user interface (UI)
+- InfluxDB HTTP API
+- `influx` CLI
```
Comment (lwandzura) [inline content/influxdb/v2.4/get-started/write.md:27]: did you want to add links to point people to CLI documentation?  same with UI and API??

File: content/influxdb/v2.4/get-started/write.md [hunk 18]
```diff
@@ -0,0 +1,391 @@
...
+- [Telegraf](/{{< latest "telegraf" >}}/)
+- {{% cloud-only %}}[MQTT](/influxdb/cloud/write-data/no-code/native-subscriptions/){{% /cloud-only %}}
+- [InfluxDB client libraries](/influxdb/v2.4/api-guide/client-libraries/)
+
+This tutorial walks through the fundamental of using **line protocol** to write
```
Comment (lwandzura) [inline content/influxdb/v2.4/get-started/write.md:32]: "walks you through"

File: content/influxdb/v2.4/get-started/query.md [hunk 19]
```diff
@@ -0,0 +1,597 @@
...
+    identifier: get-started-query-data
+weight: 102
+metadata: [3 / 5]
+---
+
+InfluxDB supports many different tools for querying data, including:
+
+- InfluxDB user interface (UI)
+- InfluxDB HTTP API
+- `influx` CLI
```
Comment (lwandzura) [inline content/influxdb/v2.4/get-started/query.md:21]: maybe add links to the docs for CLI, UI, etc?

**Review by lwandzura — COMMENTED** [review]
this looks so good!

**Pull request merged**